src/main/resources/errors.yaml

```diff
@@ -1644,6 +1644,19 @@ request-errors:
       
       Resend the command using either vector or non-vector sorting.
 
+  - scope: SORT
+    code: CANNOT_VECTOR_SORT_ON_MISMATCHED_VECTOR_DIMENSIONS
+    title: Vector sort cannot be used with mismatched vector dimensions
+    body: |-
+      The command attempted to vector sort vector columns with a mismatched vector dimension.
+
+      ${SNIPPET.VECTOR_SORT_EXPLANATION}
+      
+      The table ${keyspace}.${table} defines the vector columns: ${vectorColumns}.
```

Comment:
I think that this should only indicate problem column (not all declared, no value). But should also indicate expected and actual (wrong) vector dimensions.
This way user has more info on actual failure.